app/src/main/java/org/mozilla/fenix/settings/deletebrowsingdata/DeleteBrowsingDataFragment.kt

```diff
@@ -45,6 +45,7 @@ class DeleteBrowsingDataFragment : Fragment(R.layout.fragment_delete_browsing_da
 
         controller = DefaultDeleteBrowsingDataController(
             requireComponents.useCases.tabsUseCases.removeAllTabs,
+            requireComponents.useCases.downloadUseCases.removeAllDownloads,
```

Comment:
(This is such a small nit)
Could we make a val for `requireComponents.useCases` and `requireComponents.core` so we can just say `useCases.downloadUseCases.[...]` or `core.store`?